Resources/Maps/Shuttles/DeltaV/NTES_UCLB.yml

```diff
@@ -5365,11 +5369,15 @@ entities:
   - uid: 1180
     components:
     - type: Transform
+      anchored: False
```

Comment:
And another one

Reply:
Hey wait a second thats YOUR shuttle! >:3